### ISSUE TITLE
Public transportation not working

### ISSUE THREAD

**Issue by Donkey-Doug** [organicmaps/organicmaps#507]
Navigation using public transportation not working in the Netherlands.

**Comment by j5k** [comment]
Maybe the same as #461?

**Comment by biodranik** [comment]
No, that was another bug. Now Public Transportation works only for cities with subways. Proper public transport directions need to be developed first… Any community help on that is highly appreciated.

**Comment by marbetschar** [comment]
Would be great to have public transport for Switzerland. And to keep spam low:

@biodranik does the "public transport" feature need to be implemented on a "by country" basis - or will it be available for "all countries" once its done? If it is a "by country" feature, how should people announce interest for the feature for "their" country?

**Comment by biodranik** [comment]
The only feasible option I see now is when the community unites and helps us to integrate with existing worldwide data in OpenStreetMap. And we can help to properly add editing/updating support.
